hindi/priyanka-chopra

```diff
@@ -0,0 +1 @@
+Priyanka Chopra is an Indian film actress and singer, and the winner of the Miss World pageant of 2000. (https://en.wikipedia.org/wiki/Priyanka_Chopra)
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.